eva-server/src/main/java/uk/ac/ebi/eva/server/configuration/MongoEvaLibConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ebi.eva.server.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+import uk.ac.ebi.eva.lib.MongoConfiguration;
+
+@Configuration
+@EnableMongoRepositories("uk.ac.ebi.eva.lib.repository")
+@Import(MongoConfiguration.class)
+public class MongoEvaLibConfiguration {
```

Comment:
Not exactly, It says two things include the values in MongoEvaLibConfiguration and then enable mongo repositories. MongoConfiguration doesn't configure the repositories in eva-lib.